pilot/pkg/xds/monitoring.go

```diff
@@ -84,16 +84,16 @@ var (
 		"Pilot XDS response write timeouts.",
 	)
 
-	// Covers xds_builderr and xds_senderr for xds in {lds, rds, cds, eds}.
+	// Covers xds builder and xds sender for xds in {lds, rds, cds, eds}.
 	pushes = monitoring.NewSum(
 		"pilot_xds_pushes",
 		"Pilot build and send errors for lds, rds, cds and eds.",
 	)
 
-	cdsSendErrPushes = pushes.With(typeTag.Value("cds_senderr"))
```

Comment:
The existing name is correct. It is short type of "send errors"

Reply:
it would be a breaking change for folks who already use it in their dashboards/alerts etc. IMO it is not worth.

Reply:
Yes it is is "send err". We shouldn't change it IMO, at least not in isolation. In https://docs.google.com/document/d/1e86R7IoXVv_xFge_2iyI52f6m3BmKzBcJzf90fqd0ps/edit?usp=drive_open&ouid=109423391226753985670&resourcekey=0-X_jPaC7voBavDZhmoDxmdg I am thinking of a metrics re-haul, we shouldn't make isolated breaking changes

Reply:
ok, make sense for now.

Reply:
> Yes it is is "send err". We shouldn't change it IMO, at least not in isolation. In https://docs.google.com/document/d/1ihvByEJiWy5grWsacc6KoWBPfJ4Db9Db6yP8Lic5UXU/edit I am thinking of a metrics re-haul, we shouldn't make isolated breaking changes

I'm thinking about normalize metric name and tag, e.g. there's a lot counter metrics contains `total`, some at then, and some in the middle of the name.

Reply:
Yes I think we should do that but with consistency -- looking at all the metrics and defining a path forward (keep as is, deprecate, replace with a new one) and metrics stability levels, etc.

Our metrics currently are pretty bad, its more issues than just `total` or not